inc/saiqueue.h

```diff
@@ -80,6 +80,15 @@ typedef enum _sai_queue_attr_t
      */
     SAI_QUEUE_ATTR_INDEX = 0x00000001,
 
+    /**
+     * @brief Parent scheduler node
+     *
+     * @type sai_object_id_t
+     * @objects SAI_OBJECT_TYPE_SCHEDULER
+     * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
+     */
+    SAI_QUEUE_ATTR_PARENT_SCHEDULER_NODE = 0x00000002,
+
```

Comment:
I realized that, so added the port id as an attribute. I think it's better to add it because the queue type, port id, queue index then can be the key for this object. then scheduler node is additional attribute you need to set. 

Also parent node is a CREATE_AND_SET attribute, whereas the PORT is CREATE_ONLY attribute.